### ISSUE TITLE
[VoiceOver] Aztec Editor concerns

### ISSUE THREAD

**Issue by shiki** [wordpress-mobile/WordPress-iOS#12953]
## [A] There is an extra accessible “text field” element  

This is not the title and content text fields. It is also in between the title and content text fields in the navigation order. 

<img src="https://user-images.githubusercontent.com/198826/68908343-35162d00-0708-11ea-8ea8-9ac21088fca4.png" width="320">

## [B] “Insert {num}“ button doesn't have a hint

Something like “Inserts the selected media” can make it more obvious. 

- [x] Done in #13050 

## [C] “Insert” button should always be visible

It is more discoverable if it's there, disabled but accessible, and not suddenly appear without auditory notice. 

It can also make use of `.accessibilityValue` if it doesn't already. 

- [x] Done in #13050 

## [D] The _More_ button is dictated as “Possibly, More”. 

It should also be presented as a toggle and have the `.selected` trait when needed. A hint could also be useful here. 

We might also need to `post()` a [`.screenChanged` or `.layoutChanged` notification](https://developer.apple.com/documentation/uikit/uiaccessibility/notification/1620186-layoutchanged). 

- [x] Done in https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1246

**Comment by shiki** [comment]
> [A] There is an extra accessible “text field” element

This extra text field element is the rich content text field and is actually the correct text field. The _Share your story here..._ is just a label (placeholder) and shouldn't actually be an accessible element. 

The problem is when the rich content text is selected by VoiceOver, double-tapping it will **not activate it**. This cause seems to be because the content text field is **slightly covered** by the title text field. 

I've tried 2 things that might fix this:

1. Moving the title below the content in [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/2c8480032ef978549395f5bfa5340d932db77e68/WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift#L715-L722). This causes the title to disappear though. But the content text field can be activated by VoiceOver.
2. Changing the [constraints](https://github.com/wordpress-mobile/WordPress-iOS/blob/2c8480032ef978549395f5bfa5340d932db77e68/WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift#L668-L673) so that the content text field will not be covered by the title. This seems to work but this changes the way the scrolling behavior works. I've tried changing the insets and offsets to some degree. However, I seem to end up with more scrolling issues, like when adding images. I'd need more time to investigate this and make sure we won't get regressions. 

The odd thing is that VoiceOver users can still select and double-tap the placeholder label and they would be able to activate the content text field. 

With all these said, fixing this right now is probably too risky so I'm gonna leave it off for later if I can.
